examples/drop-rules/main.tf

```diff
@@ -0,0 +1,23 @@
+resource "chronosphere_drop_rule" "mah-drop-rule" {
```

Comment:
nit: mind getting rid of "mah" and replacing it with something else? (purely cosmetic)